tooltester/src/main/java/io/dockstore/tooltester/jenkins/JenkinsPipeline.java

```diff
@@ -0,0 +1,126 @@
+package io.dockstore.tooltester.jenkins;
+
+/**
+ * @author gluu
+ * @since 08/02/17
+ */
+
+import java.util.List;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class JenkinsPipeline {
```

Comment:
I partially generated these by getting a sample json from blue ocean and then throwing it into http://www.jsonschema2pojo.org/  .  Lots of manual modifications were made because a lot of what they generated was extremely redundant.